packages/example/pages/wallet.tsx

```diff
@@ -32,6 +32,7 @@ const account = web3.eth.accounts.privateKeyToAccount(
   'e2d7138baa3a5600ac37984e40981591d7cf857bcadd7dc6f7d14023a17b0787'
 );
 kit.addAccount(account.privateKey);
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
that's my definition `getWallet(): ReadOnlyWallet | undefined` am I out of date?

Reply:
Must be mine that's old

Reply:
If it's possible for this to be `undefined` maybe we should be handling for that case? It's not a big deal since it's the example app but just a thought.

Reply:
So, both in the tests and in the example app it's impossible.
I handle it in the example app by throwing very early on, and just ignore it in the tests.

I believe the only case it could be `undefined` is if no account was added.